### ISSUE TITLE
Fix travis validation errors

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/machine-learning#2116]
We have implemented the following linting packages within our `.travis.yml`:

```yml
...
  - pip install flake8
  - npm install -g jshint
  - gem install scss_lint
  - npm install -g bootlint
  - gem install puppet-lint
  - gem install rubocop
...
```

Now, we need to adjust the corresponding [detected offenses](https://travis-ci.org/jeff1evesque/machine-learning/builds/81566908#L1562).

**Note:** during the process of resolving this issue, additional linting processes were added (i.e. markdown, shell, etc.).  These new processes, should have been resolved in it's own dedicated issue.

**Comment by jeff1evesque** [comment]
We checked the sql database for every combination of file upload for `data_new`, and `data_append` session, inputted via the web-interface.  We also checked multiple uploads for the `data_new`, and `data_append` session.  Both test case scenarios revealed that the dataset(s) were properly uploaded to corresponding database tables.

Lastly, we manually ran our unit-test script `pytest_session.py`, and no errors were displayed.  Though, we did not manually inspect the NoSQL datastore for an updated / created model, we did manually verify that the programmatic interface properly stores dataset(s) into corresponding sql dattabase tables.

**Comment by jeff1evesque** [comment]
We will merge the current state of this issue, to the master branch.

**Comment by jeff1evesque** [comment]
This issue has not been resolved.

**Comment by jeff1evesque** [comment]
The [`multistr`](http://jshint.com/docs/options/#multistr) option will be removed during the next major release of [jshint](https://github.com/jshint/jshint), since the focus for the linter, will be code correctness, not code style.

**Comment by jeff1evesque** [comment]
1453317: Google Chrome Frame has been [retired](http://www.google.com/chromeframe/about/) since January 2014. Therefore, we should no longer use `chrome=1` in our `X-UA-Compatible` meta tag.

- https://github.com/twbs/bootlint/issues/152

**Comment by jeff1evesque** [comment]
cb3d7e7: we use the same [config](https://github.com/jscs-dev/node-jscs/blob/94f175eec822f62528e6e5ca5aab0eb1de037243/presets/airbnb.json) file that [jQuery](http://jquery.com/) implements.

**Comment by jeff1evesque** [comment]
74ab0f8: we use the same jscs [config](https://github.com/jscs-dev/node-jscs/blob/8753b05902c3531a5d9cd96c7b84c01904d55c92/presets/google.json) file that [google](https://google.com) implements.

**Comment by jeff1evesque** [comment]
After #2023 has been resolved, we need to validate corresponding puppet templates.

**Comment by jeff1evesque** [comment]
On our previous remark:

>After #2023 has been resolved, we need to validate corresponding puppet templates.

We will actually create a new issue, since this will be a new feature.

**Comment by jeff1evesque** [comment]
7135cff: we decided to [ignore](https://github.com/mivok/markdownlint/issues/114) the [MD013](https://github.com/mivok/markdownlint/blob/3c3bbece52abb85356d61a188f71d04aedc95cfb/docs/RULES.md#md013---line-length) line length rule.

**Comment by jeff1evesque** [comment]
None of the source files are compiled to corresponding asset files.
